bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4545,4 +4545,19 @@
         </sql>
     </changeSet>
 
+    <changeSet id="Create-practitioner_role-20220721" author="Abinaya, Siva, Arjun">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM provider_attribute_type where name="practitioner_role";
+            </sqlCheck>
+        </preConditions>
+        <comment>Adding provider attribute type of practitioner_role</comment>
+        <sql>
+            INSERT INTO provider_attribute_type (name, description, datatype, preferred_handler,handler_config, min_occurs, creator, date_created,
+            retired, uuid)
+            VALUES ("practitioner_role","list of practitioners",
+            "org.openmrs.customdatatype.datatype.SpecifiedTextOptionsDatatype","org.openmrs.web.attribute.handler.SpecifiedTextOptionsDropdownHandler",",Doctor,Nurse,Clinician,Paramedic", 0, 1, NOW(), 0, UUID());
```

Comment:
do we need "Clinician", "Paramedic" etc? avoid things that we don't need .. since we made it as option, we can always add later

Reply:
removed Nurse,Clinician and Paramedic..